src/shared/domain/src/main/java/org/geonetwork/domain/converter/BooleanToYN.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (c) 2003 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license,
+ * available at the root application directory.
+ */
+package org.geonetwork.domain.converter;
+
+import org.hibernate.type.CharBooleanConverter;
+
+public class BooleanToYN extends CharBooleanConverter {
```

Comment:
Even though the code is very clear and self-explanatory, I would suggest to start using Javadoc or comments on new classes, to make it easier for new developers to onboard.

Reply:
@PascalLike thanks for the review, I have add the Javadoc, please check and merge it if all looks fine.